static_test_driver/static_test_driver.ino

```diff
@@ -235,40 +234,40 @@ void loop() {
   
   BEGIN_READ
   READ_FLAG(zero_force) {
-    #if CONFIGURATION == MK_2
     Serial.println(F("Zeroing load cell"));
     scale.tare(); // Load Cell, Assuming there is no weight on the scale, reset to 0
-
-    #else
-    if (force_zero_ready) {
-      Serial.println(F("Zeroing force plate"));
-      force_zero_ready = false;
-      force_zero_val = 0;
-      for (int i = 0; i < FORCE_PLATE_NUM_HIST_VALS; i++)
-        force_zero_val += force_hist_vals[i];
-      force_zero_val /= FORCE_PLATE_NUM_HIST_VALS;
+  }
+  
+  //TODO: Update flag names to match the following code
+  READ_FLAG(zero_pressure_1) {
+    if (pressure_zero_ready_1) {
+      Serial.println(F("Zeroing fuel pressure"));
+      pressure_zero_ready_1 = false;
+      pressure_zero_val_1 = 0;
+      for (int i = 0; i < PRESSURE_NUM_HIST_VALS; i++)
+        pressure_zero_val_1 += pressure_hist_vals_1[i];
+      pressure_zero_val_1 /= PRESSURE_NUM_HIST_VALS;
     }
     else {
-      Serial.println(F("Force zero value not ready"));
+      Serial.println(F("Fuel pressure zero value not ready"));
     }
-
-    #endif
   }
-  #if CONFIGURATION == MK_2
-  READ_FLAG(zero_pressure) {
-    if (pressure_zero_ready) {
-      Serial.println(F("Zeroing pressure"));
-      pressure_zero_ready = false;
-      pressure_zero_val = 0;
+  READ_FLAG(zero_pressure_2) {
+    if (pressure_zero_ready_2) {
+      Serial.println(F("Zeroing oxidizer pressure"));
+      pressure_zero_ready_2 = false;
+      pressure_zero_val_2 = 0;
       for (int i = 0; i < PRESSURE_NUM_HIST_VALS; i++)
-        pressure_zero_val += pressure_hist_vals[i];
-      pressure_zero_val /= PRESSURE_NUM_HIST_VALS;
+        pressure_zero_val_2 += pressure_hist_vals_2[i];
+      pressure_zero_val_2 /= PRESSURE_NUM_HIST_VALS;
     }
     else {
-      Serial.println(F("Pressure zero value not ready"));
+      Serial.println(F("Oxidizer pressure zero value not ready"));
     }
   }
-  #endif
+  
+  //TODO: Tare for temperature?
```

Comment:
We probably can't (or at least don't want to) do this, since we are roughly interested in the actual temperatures here, and don't really have a point against which to zero the sensors.